pkg/rancher-desktop/config/help.ts

```diff
@@ -18,7 +18,7 @@ class HelpImpl {
 
   constructor() {
     ipcRenderer.on('get-app-version', (_event, version) => {
-      const releasePattern = /^(\d+\.\d+)\.\d+$/;
+      const releasePattern = /(\d+\.\d+)\.\d+$/;
```

Comment:
If all that can be in front of it is a `v`, then I would include that prefix in the regex instead of making it unanchored. We can make it optional, if it is not guaranteed to always be there (but why wouldn't it?):

```suggestion
      const releasePattern = /^v?(\d+\.\d+)\.\d+$/;
```